script/generate-documentation

```diff
@@ -46,6 +46,23 @@
 # target remote ($1). Target branch will be created based on
 # PULL_REQUEST_USER:gh_pages
 
+set -eo pipefail
+
+# exit status 0 if any changes detected
+anything_changed() {
+    git --no-pager diff --shortstat
+    
+    local lines_changed_in_index lines_changed_in_api other_files_changed
+
+    # it looks it is a challenge to ignore lines in git diff
+    lines_changed_in_index="$(git --no-pager diff -U0 -- index.html     | grep -v 'Last updated ' | grep -v -E '^\@\@' | grep -v -- '--- a' | grep -v '+++ b' | grep -v '^index ' | tail -n +2 | wc -l)"
```

Comment:
I'm wondering where it is possible to combine these grep invocations into a single one.